readme.md

```diff
@@ -1,69 +1,257 @@
 # Cordova Rave
 
-> A Cordova extension to add [Rave](https://www.flutterwave.com) Pay Button into your hybrid apps builds.
+ A Cordova extension to add [Rave](https://www.flutterwave.com) Pay Button into your hybrid apps builds.
 
-This module helps you easily integrate Rave into your Cordova builds. Rave lets you receive payments locally and internationally with no hassles and zero set up fees. Read more about Rave [here](https://www.flutterwave.com).
+ ![e.g](https://cloud.githubusercontent.com/assets/5229321/21958475/be1763c2-daaf-11e6-8df0-75f2e4f0168e.gif)
 
-![e.g](https://cloud.githubusercontent.com/assets/5229321/21958475/be1763c2-daaf-11e6-8df0-75f2e4f0168e.gif)
 
-### Installation
-In your app directory:
+**UPDATE (FOR IOS)**
 
-`npm install cordova-rave`
+If you are developing for IOS, ensure that the version of Xcode you are running is **10.0** (the latest version). Using a lower version causes the build to fail.
 
-### Setup
-Two steps:
+Also, do not run ```cordova prepare ios```. 
+Rather, after adding the ```ios``` platform with ```cordova platform add ios```, simply navigate to the ```ios``` folder within the ```platforms``` directory and open the ```.xcworkspace``` file in ```xcode``` then build from there.
```

Comment:
@temiadesina @Jake-parkers can you add an explanation about why building with `cordova prepare` command doesn't work for iOS? And also an explanation about why .xcworkspace fixes the issue. 
You can add links to the explanatory pages elsewhere if that is faster